extension/webpack.config.js

```diff
@@ -16,10 +16,14 @@ const config = {
       import: "./src/extension.ts",
       filename: "extension.js",
     },
-    cli: {
+    cliCreateDocumentation: {
       import: "./src/cli/CreateDocumentation.ts",
       filename: "cli/CreateDocumentation.js",
     },
+    cliRefreshXlf: {
```

Comment:
Nice catch! I totally forgot that one 😅

Reply:
I wouldn't have noticed if I hadn't decided to test `dist/*` after I saw that was what you used in the example last night. Turns out the README is quite useful :sweat_smile: